packages/amplication-client/src/Layout/SidebarHeader.tsx

```diff
@@ -11,32 +14,44 @@ type Props = {
   children: React.ReactNode;
   /**Whether to show a back button in the header of the sidebar */
   showBack?: boolean;
-  /**The URL to navigate to when the user clicks oon the back button. When empty history.goBack is used  */
+  /**The URL to navigate to when the user clicks on the back button. When empty history.goBack is used  */
   backUrl?: string;
 };
 
+const DIRECTION = "s";
+const keyMap = {
+  SUBMIT: ["CLOSE_SIDEBAR", "esc"],
+};
 const SidebarHeader = ({ children, showBack, backUrl }: Props) => {
   const history = useHistory();
 
-  function goBack() {
+  const goBack = useCallback(() => {
     if (backUrl) {
       history.push(backUrl);
     } else {
       history.goBack();
     }
-  }
+  }, [history, backUrl]);
+
+  const handlers = {
+    SUBMIT: goBack,
+  };
 
   return (
     <DrawerHeader className="side-bar__header">
+      <GlobalHotKeys keyMap={keyMap} handlers={handlers} />
```

Comment:
Why GlobalHotKeys and not HotKeys here?

Reply:
There is an issue when using GlobalHotKeys together with HotKeys so we will need to use GlobalHotKeys all the way. 
I created feature request #573